portfolio/src/main/webapp/about.html

```diff
@@ -17,10 +17,14 @@ <h1>About</h1>
 	<div class="menu">
     <nav>
         <a href="index.html"> Home </a>
-				<a href="gallery.html"> Photos </a>
+      
+		<a href="gallery.html"> Photos </a>
```

Comment:
nit: text alignment. This might be caused by cloud code editor settings -- you can match the existing files' 2-space setting with these settings (File > Settings > Open Preferences):
```
   "editor.tabSize": 2,
   "editor.useTabStops": false
```

Reply:
done!